http/src/main/java/io/micronaut/http/filter/FilterOrderProvider.java

```diff
@@ -23,7 +23,9 @@
  *
  * @author James Kleeh
  * @since 1.0
+ * @deprecated Deprecated without replacement. This class is unused and will be removed in 4.0.0.
  */
+@Deprecated
```

Comment:
```suggestion
@Deprecated( forRemoval = true)
```

Reply:
the pr is to 3.9 so we can't use forRemoval i dont think?

Reply:
true, that is Java 9 only